lint problems

the latest `main` has some lint errors:
```
fundamental-styles/stories/fn-combobox/fn-combobox.stories.js
  94:1  warning  Named exports should not use the name annotation if it is redundant to the name that would be generated by the export name  storybook/no-redundant-story-name

fundamental-styles/stories/fn-step-input/fn-step-input.stories.js
  112:1  warning  Named exports should not use the name annotation if it is redundant to the name that would be generated by the export name  storybook/no-redundant-story-name

fundamental-styles/stories/fn-textarea/fn-textarea.stories.js
  80:1  warning  Named exports should not use the name annotation if it is redundant to the name that would be generated by the export name  storybook/no-redundant-story-name

fundamental-styles/stories/fn-token/fn-token.stories.js
  74:1  warning  Named exports should not use the name annotation if it is redundant to the name that would be generated by the export name  storybook/no-redundant-story-name

✖ 4 problems (0 errors, 4 warnings)
```

--- a/stories/fn-combobox/fn-combobox.stories.js
+++ b/stories/fn-combobox/fn-combobox.stories.js
@@ -91,7 +91,6 @@ export const Combobox = () => `${localStyles}
 </div>
 `;
 
-Combobox.storyName = 'Combobox';
 Combobox.parameters = {
     docs: {
         iframeHeight: 500,

--- a/stories/fn-step-input/fn-step-input.stories.js
+++ b/stories/fn-step-input/fn-step-input.stories.js
@@ -109,7 +109,6 @@ export const StepInput = () => `${localStyles}
 </div>
 `;
 
-StepInput.storyName = 'Step Input';
 StepInput.parameters = {
     docs: {
         iframeHeight: 500

--- a/stories/fn-textarea/fn-textarea.stories.js
+++ b/stories/fn-textarea/fn-textarea.stories.js
@@ -76,5 +76,3 @@ export const Textarea = () => `${localStyles}
     </div>
 </div>
 `;
-
-Textarea.storyName = 'Textarea';

--- a/stories/fn-token/fn-token.stories.js
+++ b/stories/fn-token/fn-token.stories.js
@@ -70,5 +70,3 @@ export const Token = () => `${localStyles}
     </div>
 </div>
 `;
-
-Token.storyName = 'Token';

